testResources/SPDXJSONUnsortedIds.json

```diff
@@ -0,0 +1,250 @@
+{
+  "SPDXID" : "SPDXRef-DOCUMENT",
+  "spdxVersion" : "SPDX-2.3",
+  "creationInfo" : {
+    "comment" : "This package has been shipped in source and binary form.\nThe binaries were created with gcc 4.5.1 and expect to link to\ncompatible system run time libraries.",
+    "created" : "2010-01-29T18:30:22Z",
+    "creators" : [ "Tool: LicenseFind-1.0", "Organization: ExampleCodeInspect ()", "Person: Jane Doe ()" ],
```

Comment:
Is it intentional to have blank parentheses at the end of the Organization and Person here?

The supplier Jane Doe at Line 100 has an email address inside the parentheses.

Reply:
@bact  - I wouldn't worry too much about these inconsistencies since it is only a test file.  I hope no one really uses these as examples ;)